tensorflow/python/keras/backend.py

```diff
@@ -5817,19 +5817,28 @@ def ctc_decode(y_pred, input_length, greedy=True, beam_width=100, top_paths=1):
 
 
 @keras_export('keras.backend.map_fn')
-def map_fn(fn, elems, name=None, dtype=None):
+def map_fn(fn, elems, name=None, parallel_iterations=None, dtype=None):
```

Comment:
Can't you just use tf.map_fn instead of tf.keras.backend.map_fn instead? It seems like the keras backend just calls the public map_fn.

Reply:
Yes, right now I am using that, however I thought that `tf.keras` was going to become the "official" python interface for model creation and adding this would help people who are creating tf.keras models  that need parallel iterations

Reply:
Those can just use raw TF ops like any other piece of TF code.

Reply:
Ok, closing this PR, thanks for the feedback ;)